closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -63,10 +65,14 @@ compile-libs :
 	export \
 			$(OPENJCEPLUS_VS_LIB) \
 			GSKIT_HOME=$(OPENJCEPLUS_GSKIT_HOME) \
-			JAVA_HOME=$(OPENJCEPLUS_BOOT_JDK) \
+			JAVA_HOME=$(OPENJCEPLUS_JDK) \
 			JCE_CLASSPATH=$(OPENJCEPLUS_JCE_CLASSPATH) \
 			PLATFORM=$(OPENJCEPLUS_JGSKIT_PLATFORM) \
```

Comment:
Suggest you list these alphabetically:
```
			GSKIT_HOME=$(OPENJCEPLUS_GSKIT_HOME) \
			JAVA_HOME=$(OPENJCEPLUS_JDK) \
			JCE_CLASSPATH=$(OPENJCEPLUS_JCE_CLASSPATH) \
			$(OPENJCEPLUS_VS_LIB) \
			PLATFORM=$(OPENJCEPLUS_JGSKIT_PLATFORM) \
```
At this stage, the name `JAVA_HOME` is misleading (it isn't yet a usable JDK).

Reply:
I need to name it `JAVA_HOME` because that's what the makefile expects, given that in standalone builds we would use the java present in the system.

Reply:
Sorted alphabetically.